src/libraries/Microsoft.Extensions.Logging.EventSource/src/Microsoft.Extensions.Logging.EventSource.csproj

```diff
@@ -25,6 +25,18 @@
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Primitives\src\Microsoft.Extensions.Primitives.csproj" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <!--.NET Core 3.0 configuration is compiling some of the CodeAnalysis attributes which are nullable annotated, so adding the
+    no warn to suppress that for now.-->
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
I'm not sure I fully understand why this is necessary. Are we going to need to do this to any un-nullable annotated OOB that compiles for `netcoreapp3.0`?

Reply:
Yes. If you don’t then you get a warning saying that those attribute files are nullable annotated on a non-nullable context (since the project is not nullable). The reason why we only get this warning on the netcoreapp3.0 configuration is this line:

https://github.com/dotnet/runtime/blob/82c917aa6119c553c4fa093265c52f34476e7e13/src/libraries/Directory.Build.targets#L17

Which suppresses the warning for the other configurations.

Reply:
Can we just put `#nullable enable` in the shared src files instead? That way all the projects that include them don't need to do this?

Reply:
yeah that would work too. TBH the real fix should be that these OOBs should be annotated, but since we are not doing that in this PR, I'd be fine with adding the #nullable enable for now and log an issue to make these libraries Nullable-annotated. The only downside I see with this, is that we now won't catch if other OOBs have a netcoreapp3.0 config that need to be nullable-annotated, but since that isn't really the purpose of these files I'm fine with that approach.